package/rmt-server.spec

```diff
@@ -210,6 +185,7 @@ find %{buildroot}%{lib_dir}/vendor/bundle/ruby/*/gems/yard*/ -type f -exec chmod
 
 %files
 %attr(-,%{rmt_user},%{rmt_group}) %{app_dir}
+%exclude %{app_dir}/package-sources/
```

Comment:
@tmuntaner You don't need to exclude neither `package`, nor `package-sources` -- you are not copying `package` to `%{buildroot}/package`, everything from there goes into `%{buildroot}/%{_sysconfdir}` and the like.

Reply:
I was just trying to avoid:
```
[   45s] rmt-server.x86_64: W: files-duplicate /usr/share/rmt/package/rmt-cli_bash-completion.sh /usr/share/bash-completion/completions/rmt-cli
[   45s] 3 packages and 0 specfiles checked; 0 errors, 3 warnings.
```

in the RPMLINT report

Reply:
Hmm, that doesn't seem right, let me take a look at that.

Reply:
Whoops, sorry -- my previous comment was based on the wrong assumption that `package` isn't being included.

This copies everything from the tarball into `%{appdir}`:

https://github.com/SUSE/rmt/blob/c1e12b408ede4e0882c79cc0ec6330411f6a78ee/package/rmt-server.spec#L104

And this then includes everything in `%{appdir}` into the package, including `package`:

https://github.com/SUSE/rmt/blob/c1e12b408ede4e0882c79cc0ec6330411f6a78ee/package/rmt-server.spec#L187

So yeah, please exclude `package`.

Reply:
done in my latest commit